src/VerifyAccountHandler.ts

```diff
@@ -0,0 +1,85 @@
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { MessageCodes } from "./models/enums/MessageCodes";
+import { HttpCodesEnum } from "./models/enums/HttpCodesEnum";
+import { VerifyAccountRequestProcessor } from "./services/VerifyAccountRequestProcessor";
+import { VerifyAccountPayload } from "./type/VerifyAccountPayload";
+import { AppError } from "./utils/AppError";
+import { Constants } from "./utils/Constants";
+import { Response } from "./utils/Response";
+import { getSessionIdHeaderErrors } from "./utils/Validations";
+import { getPayloadValidationErrors } from "./utils/VerifyAccountRequestValidation";
+
+const { POWERTOOLS_METRICS_NAMESPACE = Constants.BAV_METRICS_NAMESPACE, POWERTOOLS_LOG_LEVEL = "DEBUG", POWERTOOLS_SERVICE_NAME = Constants.ACCESSTOKEN_LOGGER_SVC_NAME } = process.env;
+
+export const logger = new Logger({
+	logLevel: POWERTOOLS_LOG_LEVEL,
+	serviceName: POWERTOOLS_SERVICE_NAME,
+});
+
+const metrics = new Metrics({ namespace: POWERTOOLS_METRICS_NAMESPACE, serviceName: POWERTOOLS_SERVICE_NAME });
+
+export class VerifyAccountHandler implements LambdaInterface {
+
+	@metrics.logMetrics({ throwOnEmptyMetrics: false, captureColdStartMetric: true })
+
+	async handler(event: APIGatewayProxyEvent, context: any): Promise<Response> {
+		logger.setPersistentLogAttributes({});
+		logger.addContext(context);
+
+		try {
+			const { sessionId, body } = this.validateEvent(event);
+
+			logger.appendKeys({ sessionId });
```

Comment:
I will need to make a call to the session table to get `govuk_signin_journey_id` do you think it's worth it @ViveakV ?

Reply:
At this point we're just validating what comes into the handler so nah I don't think we need it

Reply:
But even in the processor is the same thing?

Reply:
Ah, then yeah I think worth making the call no? **govuk_signin_journey_id** is kind of our ID for the user throughout the full BE journey